ZAPD/HighLevel/HLModelIntermediette.cpp

```diff
@@ -1176,3 +1209,38 @@ void HLTerminator::OutputAssimp(aiScene* scene, std::vector<aiVector3D>* verts)
 
 	verts->clear();
 }
+
+HLSetTranslation::HLSetTranslation()
+{
+	transX = 0;
+	transY = 0;
+	transZ = 0;
+}
+
+HLSetTranslation::HLSetTranslation(float nTransX, float nTransY, float nTransZ)
+{
+	transX = nTransX;
+	transY = nTransY;
+	transZ = nTransZ;
+}
+
+HLSetTranslation::~HLSetTranslation()
+{
+}
+
+void HLSetTranslation::OutputAssimp(aiScene* scene, std::vector<aiVector3D>* verts)
+{
+	parent->lastTrans = Vec3s(transX, transY, transZ);
+}
+
+HLModelObj::HLModelObj()
+{
+}
```

Comment:
There's no need to define an empty constructor. Prefer using the `default` keyword.